packages/core/tsconfig.json

```diff
@@ -14,6 +14,8 @@
     "paths": {
       "src/*": ["src/*"],
       "@generated/*": ["@generated/*"],
+      "@faststore/core": ["index.ts"],
+      "@faststore/core/api": ["api/index.ts"],
       "@faststore/core/experimental": ["src/experimental/index.ts"]
```

Comment:
I really wanted to add comments here, but jest breaks if I do. This is needed so that when `.faststore` searches for the definitions of those imports, it uses the definitions inside the .faststore folder, not importing from node_modules (which causes build-time errors).